packages/aws-rfdk/docs/upgrade/upgrading-0.38.md

```diff
@@ -0,0 +1,254 @@
+      acmCertificate: certificate,
+      acmCertificateChain: certificateChain,
+    },
+  },
+  // ...
+});
+
+// OR
+
+const certificate = // ...your X509 RFDK certificate
+new RenderQueue(this, 'RenderQueue', {
+  // ...
+  trafficEncryption: {
+    internalProtocol: ApplicationProtocol.HTTPS,
+    externalTLS: {
+      rfdkCertificate: certificate,
+    },
+  },
+  // ...
+});
+```
+
+</details>
+
+<details><summary><b>Python</b> (click to expand)</summary>
+
+```python
+from aws_cdk.aws_elasticloadbalancingv2 import ApplicationProtocol
+from aws_rfdk.deadline import (
+    RenderQueue,
+    RenderQueueExternalTLSProps,
+    RenderQueueTrafficEncryptionProps,
+)
+
+RenderQueue(self, 'RenderQueue',
+  # no "trafficEncryption" property, so TLS is enabled both internally and externally by default
+  # ...
+)
+
+# OR
+
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    # No "internal_protocol" property, so TLS is enabled internally by default
+    # No "external_tls" property, so TLS is enabled externally by default
+  ),
+  # ...
+)
+
+# OR
+
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    # No "internal_protocol" property, so TLS is enabled internally by default
+    external_tls=RenderQueueExternalTLSProps(
+      # No "enabled" property, so external TLS will be enabled by default
+    ),
+  ),
+  # ...
+)
+
+# OR
+
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    internal_protocol=ApplicationProtocol.HTTPS,
+    # No "external_tls" property, so TLS is enabled externally by default
+  ),
+  # ...
+)
+
+# OR
+
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    internal_protocol=ApplicationProtocol.HTTPS,
+    external_tls=RenderQueueExternalTLSProps(
+      enabled=True,
+    ),
+  ),
+  # ...
+)
+
+# OR
+
+certificate = # ...your ACM certificate
+certificate_chain = # ...your ACM certificate chain
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    internal_protocol=ApplicationProtocol.HTTPS,
+    external_tls=RenderQueueExternalTLSProps(
+      acmCertificate=certificate,
+      acmCertificateChain=certificate_chain,
+    ),
+  ),
+  # ...
+)
+
+# OR
+
+certificate = # ...your X509 RFDK certificate
+RenderQueue(self, 'RenderQueue',
+  # ...
+  traffic_encryption=RenderQueueTrafficEncryptionProps(
+    internal_protocol=ApplicationProtocol.HTTPS,
+    external_tls=RenderQueueExternalTLSProps(
+      rfdk_certificate=certificate,
+    ),
+  ),
+  # ...
+)
+```
+
+</details>
+
+If you have determined that your `RenderQueue` will use HTTPS for both internal and external protocols in RFDK 0.38.0, then all you need to do to enable Deadline Secrets Management is to upgrade
+your RFDK application to use RFDK `0.38.x` and Deadline `10.1.19.x` or higher and re-deploy.
```

Comment:
What if you determine your Render Queue isn't using HTTPS? We should refer to the 0.37 upgrade doc for more detailed instructions about what will change if HTTPS is turned on.

Reply:
This has been added in the blurb from the suggestion above, so I think that should cover linking back to 0.37 upgrade docs and we don't need to add anything here. How does that sound?

Reply:
That should be fine, but reading through this again, we don't really touch on the other things to consider when enabling Secrets Management, like dedicated subnets and how to set up credentials or where to find the ones that are generated. Maybe let's add a paragraph in about that.
```suggestion
If you have determined that your `RenderQueue` will use HTTPS for both internal and external protocols in RFDK 0.38.0, 
then the next step is to decide whether you'd like to set up your own credentials used to perform admin functions for 
Secrets Management or let RFDK generate them for you, and whether you should change your VPC to use dedicated 
subnets for different constructs. More details on these considerations can be found in the 
[RFDK Deadline readme](../../lib/deadline/README.md).

Finally, all you need to do to enable Deadline Secrets Management is to upgrade
your RFDK application to use RFDK `0.38.x` and Deadline `10.1.19.x` or higher and re-deploy.
```

Reply:
Good point, added in a section with that